polyfills/Array/from/tests.js

```diff
@@ -87,35 +87,37 @@ describe('returns an array with', function () {
 			}
 		}
 
-		it('can convert from a custom iterator', function () {
-			function iterable(count) {
-				var iterator = {
-					'@@iterator': function () {
-						throw new Error();
-					}
-				};
-				iterator[Symbol.iterator] = function () {
-					var cnt = count;
-					return {
-						next: function () {
-							return cnt === 0
-								? {
-									done: true
-								}
-								: {
-									value: cnt--,
-									done: false
-								};
+		if ([][Symbol.iterator]) {
```

Comment:
Can we place this `if` statement inside the `it` function and add an `else` statement where we call `this.skip();`. This would let mocha know that a test was skipped and helps us when it comes to viewing the test reports.